content/api-reference/queries/search-activities.mdx

```diff
@@ -70,3 +74,21 @@ When specifying multiple filters, the resulting criteria is the conjuction of al
   }
 }
 ```
+
+### Pagination and sorting
+
+Pagination and sorting are optional, **though we recommend not implementing this query without pagination**. Not using pagination means that the API will return all activities at once which might cause performance issues.
+
```

Comment:
Default sorting where it is implemented is date (or date equivalent field) desc (i.e. newest to oldest). Perhaps worth mentioning! We should also mention that "The response object will contain metadata to indicate the sorting applied and the total number of records that matched all filters provided (if any).")